moviebuff.go

```diff
@@ -34,12 +37,12 @@ var (
 
 // Moviebuff allows to access to information in moviebuff using resource ids.
 type Moviebuff interface {
-	GetMovie(id string) (*Movie, error)
-	GetPerson(id string) (*Person, error)
-	GetEntity(id string) (*Entity, error)
-	GetResources(resourceType ResourceType, limit, page int) (*Resources, error)
-	GetCertifications(country string) ([]Certification, error)
-	GetHolidayCalendar(countryID string) (*Calendar, error)
+	GetMovie(ctx context.Context, id string) (*Movie, error)
```

Comment:
These are backward-incompatible changes. Any reason to go with this?

Reply:
The reason for these changes in SDK can be traced to usage of  custom spans in inbox.
Related comment: https://github.com/RealImage/qw-inbox/pull/416#discussion_r344531077

To leverage inbuilt apmhttp.WrapClient feature, we have to pass context to functions and hence the definition change.

For backward-incompatibility , should we maintain in separate branch?
Please let me know.
@sivachandran @maknahar

Reply:
I would prefer a separate set of functions. Similar to how we have all go inbuild functions.

Reply:
@sivachandran Will be changing it to v2 subpackage as discussed.